packages-web/datagrid/src/Datagrid.xml

```diff
@@ -0,0 +1,146 @@
+                    <caption>Header</caption>
+                    <description/>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Footer">
+                <property key="showFooter" type="boolean" defaultValue="false">
+                    <caption>Show footer</caption>
+                    <description/>
+                </property>
+                <property key="footerWidgets" type="widgets" required="false">
+                    <caption>Footer</caption>
+                    <description/>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Paging">
+                <property key="pageSize" type="integer" defaultValue="20">
+                    <caption>Page size</caption>
+                    <description/>
+                </property>
+                <property key="pagingEnabled" type="boolean" defaultValue="true">
+                    <caption>Page data</caption>
+                    <description/>
+                </property>
+                <property key="pagingPosition" type="enumeration" defaultValue="bottom">
+                    <caption>Position of paging bar</caption>
+                    <description/>
+                    <enumerationValues>
+                        <enumerationValue key="bottom">On the bottom</enumerationValue>
+                        <enumerationValue key="top">On top of grid</enumerationValue>
+                    </enumerationValues>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Interactions">
+                <property key="columnsSortable" type="boolean" defaultValue="true">
+                    <caption>Can sort columns</caption>
+                    <description/>
+                </property>
+                <property key="columnsFilterable" type="boolean" defaultValue="false">
+                    <caption>Can filter columns</caption>
+                    <description/>
+                </property>
+                <property key="columnsResizable" type="boolean" defaultValue="false">
+                    <caption>Can resize columns</caption>
+                    <description/>
+                </property>
+                <property key="columnsDraggable" type="boolean" defaultValue="false">
+                    <caption>Can reorder columns</caption>
+                    <description/>
+                </property>
+                <property key="columnsHidable" type="boolean" defaultValue="false">
+                    <caption>Can hide columns</caption>
+                    <description />
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Visibility">
+                <systemProperty key="Visibility"/>
+            </propertyGroup>
+            <propertyGroup caption="Common">
+                <systemProperty key="Name"/>
+                <systemProperty key="TabIndex"/>
```

Comment:
You explicitly put system properties into "Common" tab where they would end up by default anyway. Is it worth the effort?

Reply:
They will end up in a tab, this is a group :P